src/main/java/com/minecolonies/core/blocks/MinecoloniesCropBlock.java

```diff
@@ -61,19 +58,17 @@ public class MinecoloniesCropBlock extends AbstractBlockMinecolonies<Minecolonie
     private final Block preferredFarmland;
 
     private final ResourceLocation blockId;
-    private final TagKey<Biome>    preferredBiome;
 
     /**
      * Constructor to create a block of this type.
      * @param blockName the block id.
      */
-    public MinecoloniesCropBlock(final String blockName, final Block preferredFarmland, @Nullable final TagKey<Biome> preferredBiome)
+    public MinecoloniesCropBlock(final String blockName, final Block preferredFarmland)
```

Comment:
Do not remove this, we have several PRs making use of this.

Reply:
Is there really a point in leaving it in, even as a no-op? The PRs would need to adjust to this anyway by using the tags instead of the ctor arguments to have it work.

Reply:
The tags make it too complex, if you want to merely know the preferred farmland you're going to have to iterate all 4 tags to see which one it occurs in.
Not to mention this option allows crops to appear in several biome tags at the same time, which is not what this system was designed with in mind.